<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="insertArrayButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>480</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="extractMaxArrayButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>560</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Max Value</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getMaxArrayButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>620</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Max Value</string>
    </property>
   </widget>
   <widget class="QPushButton" name="insertListButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>480</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="extractMaxListButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>620</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete MAx Value</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getMaxListButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>560</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Max Value</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="insertArrayLineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>440</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="insertListLineEdit">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>440</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>1021</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>406</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Value for Bynary Array:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>406</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Value for Bynary List:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1186</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
